### DISCUSSION TITLE
Using the same files across multiple different cookiecutters organised in directories

### DISCUSSION THREAD

**Discussion by krishan-patel001** [cookiecutter/cookiecutter#1935] [Q&A]
Hi, I am attempting to use the directory structure as described here to organise multiple different cookiecutters. https://cookiecutter.readthedocs.io/en/stable/advanced/directories.html

A few of these cookiecutters need access to the same files within them e.g. a standard Makefile. Is there a way I can store this file in one place and upon generation of the cookiecutter template this file is pulled in? So I only have to update it in one place?
Currently my structure looks like this and I will have to update the Makefile in both places whenever I want a change.  

```
https://github.com/user/repo-name.git
    ├── directory1-name/
    |   ├── {{cookiecutter.project_slug}}/
                   └── Makefile
    |   └── cookiecutter.json
    └── directory2-name/
        ├── {{cookiecutter.project_slug}}/
                   └── Makefile
        └── cookiecutter.json
```

**Comment by ericof** [comment]
I also have this use case, and I'm considering how to implement this, 
Does anyone else have ideas and requests about this?
